apps/docs/remix.config.js

```diff
@@ -8,5 +8,5 @@ module.exports = {
   serverBuildDirectory: "build",
   devServerPort: 8002,
   ignoredRouteFiles: [".*"],
-  watchGlobs: ["../../packages/*/dist/index.js"],
+  watchGlobs: ["../../packages/*/dist/index.mjs"],
```

Comment:
En lei oversight fra forrige PR, men hot reloading fungerte ikke etter jeg skrev om for tree-shakeability. Fiksa det i dette PRen.